.github/workflows/build-and-test.yml

```diff
@@ -27,6 +27,8 @@ jobs:
           - 4.10.0
           - 4.11.1
           - 4.12.0
+          - 4.13.0
+          - 4.13.1
```

Comment:
Do we need to support both 4.13.x releases? That sounds like a waste of CI time when we could just check with 4.13.1. (Note for @ejgallego: independently of the present PR, I would also remove 4.06.0, 4.07.0, and 4.08.0, and at some point we could think about not testing versions older than Debian stable, which is 4.11.1.)

Reply:
I guess it'd be fine to test only the last stable version, when we added the CI we tried to test all versions due to the special nature of ppx_import accessing some internals that I think are not tested by any other CI system [neither by the OCaml upstream, right?]

Given the rate of development here, I am not so worried about the resource impact, but of course a little bit of waste in every project amounts to a large amount of carbon used :(